2021/ahmed2022re/openreview/content.tex

```diff
@@ -0,0 +1,378 @@
+\subsection{Model descriptions}
+We initially chose the ResNet-14 model because it was frequently used for image classification experiments in the original study along with the CIFAR-10 datasets. Due to the large number of trained models required for a single experiment, we did not have time to work with other models except ResNet-34, which we created ourselves to test the generalization of the claim over larger model architectures. Figure \ref{fig:pic0} shows the basic residual block of a ResNet architecture, consisting of two 3x3 convolutional layers followed by batch normalization before activation. On the same basis, Table \ref{table:table1} shows the modular architecture of ResNet-14 and ResNet-34 \cite{he2016deep} without the first 7x7 convolutional layer and the final fully-connected layer. The blocks are shown in parentheses along with the number of output channels, with the multiplier indicating the number of residual blocks in that module. 
+\begin{table}[!htb]
+\centering
+	\begin{tabular}{|c|c|c|}
+	\hline
+    Module No. & ResNet-14 & ResNet-34\\
+	\hline
+	& & \\
+		1 & \[\begin{bmatrix} 3 \times 3, & 16 \\ 3 \times 3, & 16 \end{bmatrix} \] \times 2 & \[\begin{bmatrix} 3 \times 3, & 64 \\ 3 \times 3, & 64 \end{bmatrix} \] \times 3 \\
+		& & \\
+		\hline
+		& & \\
+		 2 & \[\begin{bmatrix} 3 \times 3, & 32 \\ 3 \times 3, & 32 \end{bmatrix} \] \times 2 & \[\begin{bmatrix} 3 \times 3, & 128 \\ 3 \times 3, & 128 \end{bmatrix} \] \times 4  \\
+		 & & \\
+		 \hline
+		 & & \\
+		3 & \[\begin{bmatrix} 3 \times 3, & 64 \\ 3 \times 3, & 64 \end{bmatrix} \] \times 2 & \[\begin{bmatrix} 3 \times 3, & 256 \\ 3 \times 3, & 256 \end{bmatrix} \] \times 6 \\
+		& & \\
+		 \hline
+		 & & \\
+		4 &  & \[\begin{bmatrix} 3 \times 3, & 512 \\ 3 \times 3, & 512 \end{bmatrix} \] \times 3 \\
+		 & & \\
+		\hline
+	\end{tabular}
+	\caption{Architecture of ResNet-14 and ResNet-34 in modular format}
+	\label{table:table1}
+\end{table}
+
+\begin{figure}[!htb]
+	\centering
+	\includegraphics[width=0.25\textwidth]{Capture.png}
```

Comment:
This needs to be a relative import. Change to `../openreview/Capture.png` to make it work.